tests/test_schema.py

```diff
@@ -2048,3 +2048,38 @@ def test_dump_only(self, schema, data):
         assert 'str_dump_only' not in result.data
         assert 'str_load_only' in result.data
         assert 'str_regular' in result.data
+
+
+class TestStrictDefault:
+
+    class SchemaTrueByMeta(Schema):
+        class Meta:
+            strict = True
+
+    class SchemaFalseByMeta(Schema):
+        class Meta:
+            strict = False
+
+    class SchemaWithoutMeta(Schema):
+        pass
+
+    def test_default(self):
+        assert self.SchemaWithoutMeta().strict is False
+
+    def test__meta_true(self):
```

Comment:
1. I think it is safe and more common not to use double underscores in test names.
2. You miss two tests when you use SchemaWithoutMeta and override with init argument (both True and False).